[PATCH] set LIB_SUBDIR to libdir

Upstream-Status: Inappropriate [configuration]

LIB_SUBDIR is lib for all mips64 cpu, but it is wrong, the application of
MIPS64 is able to 64bit, or 32bit, so setting lib/lib64 to LIB_SUBDIR is
not suitable. 
---
 CMakeLists.txt | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e9259a6..cb10a24 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -250,10 +250,7 @@ ENDIF(WIN32)
 SET(EXECUTABLE_OUTPUT_PATH "${CMAKE_BINARY_DIR}/bin")
 SET(LIBRARY_OUTPUT_PATH "${CMAKE_BINARY_DIR}/lib")
 
-SET(LIB_SUBDIR "lib")
-IF(RPM_BUILD AND CMAKE_SYSTEM_PROCESSOR MATCHES "(x86_64|ppc64|ppc64p7|s390x|sparc64)") 
-  SET(LIB_SUBDIR "lib64")
-ENDIF()
+SET(LIB_SUBDIR "${BASE_LIBDIR}")
 
 MESSAGE(STATUS "Installation library subdir: ${LIB_SUBDIR}")
 
-- 
1.9.1

